### PR TITLE
Setup Working Environment

### PR DESCRIPTION
**Pull request by Uyoyo-W**
Uploaded "index.html", "style.css" and "assets" folders

### CODE REVIEW

**Review by devRaphe — APPROVED** [review]
LGTM

**Review by devRaphe — COMMENTED** [review]
@HanaMK please review this and merge. thanks

**Review by Hana-Kari — COMMENTED** [review]
Looks good

**Review by Hana-Kari — COMMENTED** [review]
It's Ok

File: style.css/style-guide.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+COLORS
```
Comment (Hana-Kari) [inline style.css/style-guide.md:1]: It's Ok

File: index.html/index-guide.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```
Comment (Hana-Kari) [inline index.html/index-guide.html:1]: It's Ok

**Review by Hana-Kari — COMMENTED** [review]
Well done

**Pull request merged**